test/integration/aws/default/build/config.tf

```diff
@@ -33,4 +33,101 @@ output "role_for_config_recorder_arn" {
 
 output "config_recorder_name" {
   value = "${aws_config_configuration_recorder.config_recorder.name}"
-}
\ No newline at end of file
+}
+
+#======================================================#
+#             Configuration Delivery Channel
+#======================================================#
+resource "aws_config_delivery_channel" "delivery_channel_01" {
+  name           = "delivery_channel_01"
+  s3_bucket_name = "${aws_s3_bucket.bucket_for_delivery_channel.bucket}"
+  depends_on     = ["aws_config_configuration_recorder.config_recorder_for_delivery_channel"]
+  sns_topic_arn  = "${aws_sns_topic.sns_topic_for_delivery_channel.arn}"
+  snapshot_delivery_properties = {
+    delivery_frequency = "TwentyFour_Hours"
+  }
+}
+
+resource "aws_config_configuration_recorder" "config_recorder_for_delivery_channel" {
```

Comment:
It looks like the default AWS account limit for config recorders is 1 - so when I run the integration tests, it fails here.  I think we'll have to re-use the one defined for the `aws_config_recorder` tests.

```
Error applying plan:

1 error(s) occurred:

* aws_config_configuration_recorder.config_recorder_for_delivery_channel: 1 error(s) occurred:

* aws_config_configuration_recorder.config_recorder_for_delivery_channel: Creating Configuration Recorder failed: MaxNumberOfConfigurationRecordersExceededException: Failed to put configuration recorder 'config_recorder_for_delivery_channel' because the maximum number of configuration recorders: 1 is reached.
	status code: 400, request id: 47d66458-2dea-11e8-a668-9734f608a028
```